src/test/java/networkbook/logic/parser/NetworkBookParserTest.java

```diff
@@ -139,6 +140,12 @@ public void parseCommand_help() throws Exception {
         assertTrue(parser.parseCommand(HelpCommand.COMMAND_WORD + " 3") instanceof HelpCommand);
     }
 
+    @Test
+    public void parseCommand_save() throws Exception {
+        assertTrue(parser.parseCommand(SaveCommand.COMMAND_WORD) instanceof SaveCommand);
+        assertTrue(parser.parseCommand(SaveCommand.COMMAND_WORD + " 3") instanceof SaveCommand);
+    }
+
```

Comment:
Would be better to use `equals` method and override the method in `SaveCommand`
rather than hardcoding the logic of `equals` here.

Reply:
Okay will change it